sdk/push-fcm/src/main/java/com/klaviyo/pushFcm/Extensions.kt

```diff
@@ -0,0 +1,213 @@
+package com.klaviyo.pushFcm
```

Comment:
This file adds extension properties to `RemoteMessage` as a nice shortcut for parsing all our data keys